packages/ui/components/StickyHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import classnames from 'classnames'
+import { PropsWithChildren } from 'react'
+
+export interface StickyHeaderProps
+  extends React.DetailedHTMLProps<
+      React.HTMLAttributes<HTMLElement>,
+      HTMLElement
+    >,
+    PropsWithChildren {
+  variant?: 'primary' | 'light' | 'dark'
```

Comment:
we don't need a variant for this. tailwind has it's own `dark:` handler. we can just utilize that prefix in the future.